docs/source/performance/caching.md

```diff
@@ -331,14 +331,21 @@ The effect of setting `honorSubgraphCacheControlHeader` to `false` is to have no
 
 ## Caching with a CDN
 
-Whenever Apollo Server sends an operation response that has a non-zero `maxAge`, it includes a `Cache-Control` HTTP header that describes the response's cache policy.
+Whenever Apollo Server sends an operation response that is cacheable, it includes a `Cache-Control` HTTP header that describes the response's cache policy.
```

Comment:
@glasser Additionally, it now also sets the `cache-control` header when it's not cacheable as well. We do go on to clarify this below, but the way this is written leads me to think it's only set when cacheable.